examples/src/integrationTest/java/org/apache/ignite/example/storage/ItPageMemoryStorageExampleTest.java

```diff
@@ -58,16 +61,20 @@ public void testInMemoryExample() throws Exception {
     }
 
     private void addVolatileDataRegionConfig(String name) throws Exception {
```

Comment:
Removed these methods. Because after some discussions we found out, that runtime update of storage profiles is redundant and harmful feature. Node will not provide the new information about supported storage profiles to cluster after this type of update without restart (as with other node attributes as well).